From 0e9072b775625258071bf86a66e192877e62d132 Mon Sep 17 00:00:00 2001
From: Thao Luong <thao.luong.uw@rvc.renesas.com>
Date: Wed, 27 May 2015 11:28:28 +0700
Subject: [PATCH 12/25] gstomx: Set the used flag of the GstOMXBuffer if the 
 buffer is used by the port

This flag is used to check if the buffer is used by the port.

refs #69206
Signed-off-by: Thao Luong <thao.luong.uw@rvc.renesas.com>
---
 omx/gstomx.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/omx/gstomx.c b/omx/gstomx.c
index 373ecb2..59b1dc0 100644
--- a/omx/gstomx.c
+++ b/omx/gstomx.c
@@ -2035,6 +2035,8 @@ gst_omx_port_populate_unlocked (GstOMXPort * port)
        */
       buf->omx_buf->nFlags = 0;
 
+      buf->used = TRUE;
+
       err = OMX_FillThisBuffer (comp->handle, buf->omx_buf);
 
       if (err != OMX_ErrorNone) {
-- 
2.4.0

